plugins/rapid7_intsights/plugin.spec.yaml

```diff
@@ -386,7 +386,7 @@ actions:
       is_closed:
         title: Closed Status
         description: Status of the alert, either closed or open
-        type: boolean
+        type: string
```

Comment:
is there a significance to changing this from boolean type to a string? the is_closed name implies a true or false expected value

Reply:
This will have to be a major version bump unfortunately but should go through